cinema_service/settings.py

```diff
@@ -127,14 +130,13 @@
 
 USE_TZ = False
```

Comment:
Consider setting USE_TZ to True to enable timezone support, which is generally recommended for handling datetime objects in a consistent manner across different environments.